packages/wallet/backend/src/card/service.ts

```diff
@@ -131,11 +135,13 @@ export class CardService {
     token: string,
     cypher: string
   ): Promise<void> {
-    await this.ensureAccountExists(userId, cardId)
+    const cardAccount = await this.ensureAccountExists(userId, cardId)
 
     await this.gateHubClient.changePin(token, cypher)
 
     await User.query().findById(userId).patch({ isPinSet: true })
+
+    await this.fundAccount(cardAccount)
```

Comment:
This should be done at sign up instead of doing it when we fetch the cards.

Reply:
it is on set pin